Bottom: cbebfa2b63cb7d5c56a0a3c79e43af01e9711edd
Top:    4fbb629f3be393f89d05c6a4b2cae5949f4b004f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-06 16:00:33 -0500

Create swarm


---
diff --git a/roles/docker-swarm/defaults/main.yml b/roles/docker-swarm/defaults/main.yml
new file mode 100644
index 00000000..e69de29b
diff --git a/roles/docker-swarm/tasks/main.yml b/roles/docker-swarm/tasks/main.yml
new file mode 100644
index 00000000..fed0e296
--- /dev/null
+++ b/roles/docker-swarm/tasks/main.yml
@@ -0,0 +1,6 @@
+---
+- name: Initialize Swarm
+
+- name: Initialize Manager Node
+
+- name: Initialize Worker Node
